assets/js/mbta.ts

```diff
@@ -2107,7 +2107,7 @@ const stationConfig: {
             modes: {
               auto: false,
               custom: true,
-              headway: false,
+              headway: true,
```

Comment:
We shouldn't allow this mode on this sign, since it's a mezzanine.

Reply:
Looks like WTC also has a mezzanine sign so I set that to false as well.